src/client/client.ts

```diff
@@ -324,8 +324,19 @@ export class _ClientImpl<
 
     this.store = createStore(this.reducer, this.initialState, enhancer);
 
-    if (!multiplayer) multiplayer = DummyTransport;
-    this.transport = multiplayer({
+    let multiplayerTransport: (opts: TransportOpts) => Transport;
+
+    if (!multiplayer) {
+      multiplayerTransport = DummyTransport;
+    } else if (typeof multiplayer === 'string' && multiplayer === 'cartesify') {
```

Comment:
suggestion: triple equal operator already checks for type, so the check for typeof "string" shouldn't be necessary